src/Server.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Middleware\LimitConcurrentRequestsMiddleware;
+use React\Http\Middleware\RequestBodyBufferMiddleware;
+use React\Http\Middleware\RequestBodyParserMiddleware;
+use React\Socket\ServerInterface;
+
+/**
+ * Facade around StreamingServer with sane defaults for 80% of the use cases.
+ * The requests passed to your callable are fully compatible with PSR-7 because
+ * the body of the requests are fully buffered and parsed, unlike StreamingServer
+ * where the body is a raw ReactPHP stream.
+ *
+ * Wraps StreamingServer with the following middleware:
+ * - RequestBodyBufferMiddleware
+ * - RequestBodyParserMiddleware
+ * - The callable you in passed as first constructor argument
```

Comment:
`LimitConcurrentRequestsMiddleware` missing now here.

Reply:
Fixed that :+1: